tests/nimbleclientbase.py

```diff
@@ -0,0 +1,114 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+
+# @author alokranjan
+
+"""Test base class for Nimos Client."""
+
+from nimbleclient.v1 import client
+import datetime
+import time
+import configparser
+import os
+import threading
+
+
+# the below variable "SKIPTEST" is to be used if a user wants to just run one
```

Comment:
nit: The

Reply:
have renamed to Below variable. removed "the" and renamed below to Below